packages/editor/src/components/panel-color/index.js

```diff
@@ -17,6 +18,12 @@ import withColorContext from '../color-palette/with-color-context';
 import { getColorObjectByColorValue } from '../colors';
 
 function PanelColor( { colors, title, colorValue, initialOpen, ...props } ) {
+	deprecated( 'wp.editor.PanelColor', {
```

Comment:
Here we should have wp.components.PanelColor.

Reply:
Fixed!